Assets/MixedRealityToolkit-SDK/Features/Audio/Influencers/AudioInfluencerController.cs

```diff
@@ -0,0 +1,313 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Core.Utilities;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.SDK.Audio.Influencers
+{
+    /// <summary>
+    /// Class which supports components implementing <see cref="IAudioInfluencer"/> being used with audio sources.
+    /// </summary>
+    /// <remarks>
+    /// AudioInfluencerController requires an <see cref="AudioSource"/> component. If one is not attached, it will be added automatically.
+    /// Each sound playing game object needs to have an AudioInfluencerController attached in order to have it's audio influenced.
+    /// </remarks>
+    [RequireComponent(typeof(AudioSource))]
+    [DisallowMultipleComponent]
+    public class AudioInfluencerController : MonoBehaviour
+    {
+        /// <summary>
+        /// Frequency below the nominal range of human hearing.
+        /// </summary>
+        /// <remarks>
+        /// This frequency can be used to set a high pass filter to allow all
+        /// audible frequencies through the filter.
+        /// </remarks>
+        public static readonly float NeutralLowFrequency = 10.0f;
```

Comment:
const?

Reply:
we have discussed this before. we agreed on this for HTK. why change it now?

Reply:
Link?

I'm not sure that this is something we'll ever change, so I think const is okay here.

Again I'll leave it up to your discretion.